### PR TITLE
Added more tasks.

### PR DESCRIPTION
**Pull request by sdhutchins**
- Added details on getting set up for Jetstream2 and Cheaha
- Added better wording.

### CODE REVIEW

**Comment by bjoyce3** [comment]
Replace 

- [ ] Attend workshops to become ready for the hackathon.
    - [ ] Computing Resources Workshop: An Introduction to Jetstream2, HPC, and Git/GitHub | Register at https://tinyurl.com/mubxwc8f | July 27 at 2PM
    - [ ] U-BRITE Bootcamp: Learn how to use PAGER & SEAS for multi-omics downstream analysis by Dr. Jake Chen | Register at https://tinyurl.com/yxx23fph | 

With (URL/Box/YouTube) links to videos from those sessions. They've already passed.

**Comment by sdhutchins** [comment]
> Replace
> 
> * [ ]  Attend workshops to become ready for the hackathon.
>   
>   * [ ]  Computing Resources Workshop: An Introduction to Jetstream2, HPC, and Git/GitHub | Register at https://tinyurl.com/mubxwc8f | July 27 at 2PM
>   * [ ]  U-BRITE Bootcamp: Learn how to use PAGER & SEAS for multi-omics downstream analysis by Dr. Jake Chen | Register at https://tinyurl.com/yxx23fph |
> 
> With (URL/Box/YouTube) links to videos from those sessions. They've already passed.

Awesome. Will do. 😄

**Comment by tkmamidi** [comment]
Is it too late to split the Leader and member checklists? It'll be easy for us and the team leaders to send email/notification and gather responses?

**Comment by sdhutchins** [comment]
> Is it too late to split the Leader and member checklists? It'll be easy for us and the team leaders to send email/notification and gather responses?

I'm working on the team member checklist - it'll be in the main repo. I'm going to push there tonight.

**Comment by sdhutchins** [comment]
@bjoyce3 Should be ready now.

File: README.md [hunk 1]
```diff
@@ -22,14 +22,28 @@ We'd like to welcome you to Hackin' Omics 2022! We hope you are as excited as we
...
-    - [ ] Computing Resources Workshop: An Introduction to Jetstream2, HPC, and Git/GitHub | Register at https://tinyurl.com/mubxwc8f | July 27 at 2PM
-    - [ ] U-BRITE Bootcamp: Learn how to use PAGER & SEAS for multi-omics downstream analysis by Dr. Jake Chen | Register at https://tinyurl.com/yxx23fph | July 29 at 10AM
+    - [ ] Add your team members to the [U-BRITE organization](https://github.com/orgs/u-brite/teams/hackin-omics/members) if they have a GitHub account or have them create an account.
+- [ ] Attend workshops or watch recordings to become ready for the hackathon.
+    - [ ] [Computing Resources Workshop: An Introduction to Jetstream2, HPC, and Git/GitHub](https://uab.box.com/s/ddsxtndjci74uob55xil6qwcrp6ksjjl) by [Dr. Blake Joyce](https://github.com/bjoyce3)
+    - [ ] [U-BRITE Bootcamp: Learn how to use PAGER & SEAS for multi-omics downstream analysis]() [by Dr. Jake Chen](https://github.com/jakechen1)
 - [ ] Ensure any datasets that need to be dowloaded have been mentioned by communicating with Dr. Blake Joyce via slack (His slack is `@Blake Joyce (UAB Research Computing)` or tagging the technical committee on GitHub using `@u-brite/technical-committee`.
```
Comment (tkmamidi) [inline README.md:36]: Downloading datasets is all Zhandos.

File: README.md [hunk 2]
```diff
@@ -22,14 +22,28 @@ We'd like to welcome you to Hackin' Omics 2022! We hope you are as excited as we
...
-  - [ ] Submit this [form](https://airtable.com/shrnpxEogyAzlRWGX) to submit any large dataset that isn't available via an API or is too large to be hosted on a laptop.
+  - [ ] Submit this [form](https://airtable.com/shrnpxEogyAzlRWGX) to submit any datasets or tools you'll be using throughtout the hackathon.
+- [ ] Determine if you’ll need cloud ([Jetstream2](https://jetstream-cloud.org/about/index.html)) or HPC computing (Cheaha) for the hackathon. Contact Blake Joyce in slack to discuss your team’s needs or discuss in the #team-leaders  channel on slack. 
+  - [ ] If you want access to cloud (Jetstream2):
+    - Jetstream 2 cloud computing (free cloud option):
+    - Please create an [XSEDE account](https://portal.xsede.org/my-xsede?p_p_id=58&p_p_lifecycle=0&p_p_state=maximized&p_p_mode=view&_58_struts_action=%2Flogin%2Fcreate_account). 
+    - Have your team members create an XSEDE account.
+    - Gather your and all your team members’ XSEDE accounts and email to [Blake Joyce](bjoyce@uab.edu) in a single list.
```
Comment (tkmamidi) [inline README.md:43]: Email or slack is fine, I think

File: resources/README.md [hunk 3]
```diff
@@ -8,12 +8,15 @@ Your main job is to identify key project skill requisites, acquire the datasets
 
 - Propose a project proposal or select to take the lead on a proposal.
 - Select datasets that fits the scope of the hackathon.
-- Coordinate with our technical team, [Dr. Blake Joyce](), to ensure any data that needs dowloading can be done prior to the hackathon.
+- Coordinate with our technical team, [Dr. Blake Joyce](bjoyce@uab.edu), to ensure any data that needs dowloading can be done prior to the hackathon.
```
Comment (tkmamidi) [inline resources/README.md:11]: Zhandos here again

File: README.md (hunk 1 above)
Comment (sdhutchins) [reply README.md:36]: Only had Blake since he's the technical lead - 1 point of contact. But don't mind adding Zhandos.

File: README.md (hunk 2 above)
Comment (sdhutchins) [reply README.md:43]: He wrote this lol

File: README.md (hunk 1 above)
Comment (tkmamidi) [reply README.md:36]: Lets leave it to Zhandos. Blake is reserved for getting access to stuff.

File: README.md (hunk 2 above)
Comment (tkmamidi) [reply README.md:43]: Cool. Works for me.

File: README.md [hunk 4]
```diff
@@ -22,14 +22,28 @@ We'd like to welcome you to Hackin' Omics 2022! We hope you are as excited as we
...
+    - [ ] Your repository should be PUBLIC.
     - [ ] Refine your project proposal based on the feedback given (if there was feedback).
-- [ ] Attend workshops to become ready for the hackathon.
-    - [ ] Computing Resources Workshop: An Introduction to Jetstream2, HPC, and Git/GitHub | Register at https://tinyurl.com/mubxwc8f | July 27 at 2PM
-    - [ ] U-BRITE Bootcamp: Learn how to use PAGER & SEAS for multi-omics downstream analysis by Dr. Jake Chen | Register at https://tinyurl.com/yxx23fph | July 29 at 10AM
+    - [ ] Add your team members to the [U-BRITE organization](https://github.com/orgs/u-brite/teams/hackin-omics/members) if they have a GitHub account or have them create an account.
+- [ ] Attend workshops or watch recordings to become ready for the hackathon.
+    - [ ] [Computing Resources Workshop: An Introduction to Jetstream2, HPC, and Git/GitHub](https://uab.box.com/s/ddsxtndjci74uob55xil6qwcrp6ksjjl) by [Dr. Blake Joyce](https://github.com/bjoyce3)
+    - [ ] [U-BRITE Bootcamp: Learn how to use PAGER & SEAS for multi-omics downstream analysis]() [by Dr. Jake Chen](https://github.com/jakechen1)
```
Comment (tkmamidi) [inline README.md:35]: add this link to the bootcamp since we don't have a video - https://github.com/u-brite/u-brite-bootcamp

**Comment by sdhutchins** [comment]
Should be finished. Please review @bjoyce3 or @tkmamidi

**Review by tkmamidi — APPROVED** [review]

**Pull request merged**